iphone/Classes/TiUIApplicationShortcutsProxy.m

```diff
@@ -211,6 +212,14 @@ - (UIApplicationShortcutIcon *)findIcon:(id)value
     return [UIApplicationShortcutIcon iconWithTemplateImageName:[self urlInAssetCatalog:value]];
   }
 
+#ifdef IS_SDK_IOS_13
+  if ([value isKindOfClass:[TiBlob class]] && [TiUtils isIOSVersionOrGreater:@"13.0"]) {
```

Comment:
do you need to check the iOS version here if you have the ifdef guard as well?

Reply:
Yes. Its needed. UIApplicationShortcutIcon's api 'iconWithSystemImageName' is not available for iOS < 13. Let's say I ran my app using sdk 8.2.1 with Xcode 11 and run on iOS 12, in that case it will give error at run time.